README.md

```diff
@@ -1 +1,76 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> n this project, I will set up a "Hello world" repository. No complex coding is required for this project . My goal here is to master all of the tools git and git hup  
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages:Html and CSS
+- Technologies used :GIT and GitHup 
+
+
```

Comment:
- [ ] This area should contain information on how someone can get a copy of the project and run it on his/her machine. Take a look again at [this](https://github.com/microverseinc/readme-template) link to see the steps on getting started.